test/Feature/SilentKleeAssume.c

```diff
@@ -1,30 +0,0 @@
-// RUN: %clang -emit-llvm -g -c -o %t.bc %s
```

Comment:
I would keep these tests. Maybe your new ones subsume them, but it is less obvious that SilentKleeAssume is tested there.